src/Planner/PlannerJoinTree.cpp

```diff
@@ -1318,6 +1318,14 @@ JoinTreeQueryPlan buildQueryPlanForJoinNode(const QueryTreeNodePtr & join_table_
                 std::swap(table_join_clause.key_names_right.at(asof_condition.key_index), table_join_clause.key_names_right.back());
             }
         }
+
+        if (join_clauses_and_actions.full_join_expressions_actions)
```

Comment:
Maybe we can rename this expressions and method to `mixed` instead of full to have more unoform naming?